python/feast_spark/pyspark/launcher.py

```diff
@@ -484,6 +485,46 @@ def get_job_by_id(job_id: str, client: "Client") -> SparkJob:
     return launcher.get_job_by_id(job_id)
 
 
+def get_health_metrics(
+    client: "Client", project: str, table_names: List[str],
+) -> Dict[str, List[str]]:
+    all_redis_keys = [f"{project}:{table}" for table in table_names]
+    metrics = client.metrics_redis.mget(all_redis_keys)
+
+    valid_feature_tables = []
+    failed_feature_tables = []
+
+    for metric, name in zip(metrics, table_names):
+        feature_table = client.feature_store.get_feature_table(
+            project=project, name=name
+        )
+        max_age = feature_table.max_age
+        # Only perform ingestion health checks for Feature tables with max_age
+        if not max_age:
+            valid_feature_tables.append(name)
+            continue
+
+        # If there are missing metrics in Redis; None is returned if there is no such key
+        if not metric:
+            failed_feature_tables.append(name)
+
```

Comment:
Should add a continue here. Otherwise the execution will proceed and throw exception when trying to load None as json string